addons/mail/wizard/mail_compose_message.py

```diff
@@ -125,6 +126,16 @@ def _get_composition_mode_selection(self):
     message_type = fields.Selection(default="comment")
     subtype_id = fields.Many2one(default=lambda self: self.sudo().env.ref('mail.mt_comment', raise_if_not_found=False).id)
 
+    is_default_template = fields.Boolean(compute='_compute_is_default_template')
+
+    @api.depends('template_id')
+    def _compute_is_default_template(self):
+        IrValues = self.env['ir.values'].sudo()
+        for compose in self:
+            template_id = IrValues.get_default('mail.compose.message', 'template_id', condition='mail_compose_act_btn=' + (self.env.context.get('mail_compose_act_btn') or '') + '&' + 'model=' + (compose.model or ''))
```

Comment:
for code readability, maybe extract the computing of `condition=` to an helper function

Reply:
same issue with `ir.values` here